tests/services/test_language_config.py

```diff
@@ -32,7 +33,7 @@ def test_that_language_config_service_treats_missing_templates_as_unsupported_la
 ):
     mock_data_loader.side_effect = ValueError
     with pytest.raises(LanguageNotSupportedError):
-        language_config_service.get_language_config("BadLang")
+        language_config_service.get_language_config("BadLang", True)
```

Comment:
Can you include the argument here for readability (and below)?
```suggestion
        language_config_service.get_language_config("BadLang", include_linter=True)
```